telegram/ext/_application.py

```diff
@@ -1223,7 +1307,7 @@ async def dispatch_error(
                     job=job,
                     coroutine=coroutine,
                 )
-                if not block or (
+                if not block or (  # If error handler has `block=False`, create a Task to run cb
```

Comment:
cb = ?

Reply:
callback. cb is used all over updater too

Reply:
ok, makes sense